### ISSUE TITLE
Lazy loading

### ISSUE THREAD

**Issue by pavelloz** [giscus/giscus#431]
Hello,

What do you think about adding loading="lazy" to the iframe to possibly save transfer or people visiting page but not going to till the end (where usually comments live)?

Read more:
https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading

**Comment by LITUATUI** [comment]
Good idea!

**Comment by laymonage** [comment]
Implemented in #438.

Add `data-loading="lazy"` to your `<script>` tag to enable this.

```html
<script src="https://giscus.app/client.js"
        ...
        data-loading="lazy"
        ...>
</script>
```
